### PR TITLE
Cloud Talent solution v4 PART 1

### PR DESCRIPTION
**Pull request by munkhuushmgl**

### CODE REVIEW

**Comment by amanda-tarafa** [comment]
@munkhuushmgl I probably won't get to this until next week. Let's chat later if you need me to take a look before that.

**Comment by munkhuushmgl** [comment]
> @munkhuushmgl I probably won't get to this until next week. Let's chat later if you need me to take a look before that.

Thats okay, this is not urgent.

File: jobs/api/v4/JobsSample/CreateCompanySample.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            string parent = TenantName.Format(projectId, tenantId);
+            Company company = new Company
+            {
+                DisplayName = displayName,
+                ExternalId = externalId
+            };
+
+            CreateCompanyRequest request = new CreateCompanyRequest
+            {
+                Parent = parent,
```
Comment (amanda-tarafa) [inline jobs/api/v4/JobsSample/CreateCompanySample.cs]: nit: Is there something like `ParentAsParentName`? Then you can use:
(on line 26) `ParentName parent = TenantName.From(projectId, tenantId);`

(on line 35) `ParentAsParentName = parent;`

Comment (munkhuushmgl) [reply jobs/api/v4/JobsSample/CreateCompanySample.cs]: I have changed all string values into Specific Objects [TenantName, ProjectName, ParentAsTenantOrProjectNameOneof ]

**Pull request merged**